moveit_core/kinematics_base/include/moveit/kinematics_base/kinematics_base.h

```diff
@@ -59,7 +59,7 @@ MOVEIT_CLASS_FORWARD(RobotModel)
 namespace kinematics
 {
 // Logger
-rclcpp::Logger LOGGER = rclcpp::get_logger("kinematics_base");
+static rclcpp::Logger LOGGER_KINEMATICS_BASE = rclcpp::get_logger("kinematics_base");
```

Comment:
Why is this static? It is not in a method or a class so when would their ever be multiple copies?

Reply:
I think I made this static (quick fix) because it used in the cpp and the hpp and otherwise was failing. Suggestions?